docs/architecture/gas/estimator.md

```diff
@@ -20,100 +20,9 @@ clients.
-```
-
-You might also want to run a hardware-agnostic estimation using the following
-command. It uses `docker` and `qemu` under the hood, so it will be quite a bit
-slower. You will need to install `docker` to run this command.
-
-
-```bash
-cargo run --release -p runtime-params-estimator --features required -- \
-    --accounts-num 20000 --additional-accounts-num 2000000 \
-    --iters 3 --warmup-iters 1 --metric icount --docker --full \
-    --costs=ActionReceiptCreation,ActionTransfer,ActionCreateAccount,ActionFunctionCallBase
-```
-
-Note how the output looks a bit different now. The `i`, `r` and `w` values show
-instruction count, read IO bytes, and write IO bytes respectively. The IO byte
-count is known to be inaccurate.
-
-```
-+ /host/nearcore/runtime/runtime-params-estimator/emu-cost/counter_plugin/qemu-x86_64 -plugin file=/host/nearcore/runtime/runtime-params-estimator/emu-cost/counter_plugin/libcounter.so -cpu Westmere-v1 /host/nearcore/target/release/runtime-params-estimator --home /.near --accounts-num 20000 --iters 3 --warmup-iters 1 --metric icount --costs=ActionReceiptCreation,ActionTransfer,ActionCreateAccount,ActionFunctionCallBase --skip-build-test-contract --additional-accounts-num 0 --in-memory-db
-ActionReceiptCreation         214_581_685_500 gas [  1716653.48i 0.00r 0.00w]     (computed in 6.11s) 
-ActionTransfer                 21_528_212_916 gas [   172225.70i 0.00r 0.00w]     (computed in 4.71s) 
-ActionCreateAccount            26_608_336_250 gas [   212866.69i 0.00r 0.00w]     (computed in 4.67s) 
-ActionFunctionCallBase         12_193_364_898 gas [    97546.92i 0.00r 0.00w]     (computed in 2.39s) 
-
-
-Finished in 17.92s, output saved to:
-
-    /host/nearcore/costs-2022-11-01T16:27:36Z-e40863c9b.txt
-```
-
-The difference between the metrics will be discussed in the
-section on [Estimation Metrics](#estimation-metrics).
-
-You should now be all setup for running estimations on your local machine. Also
-check `cargo run -p runtime-params-estimator --features required -- --help` for
-the list of available options.
-
-## From Estimations to Parameter Values
-
-To calculate the final gas parameter values, there is more to be done than just
-running a single command. After all, these parameters are part of the protocol
-specification. They cannot be changed easily. And setting them to a wrong value
-can cause severe system instability.
-
-Our current strategy is to run estimations
-with two different metrics and do so on standardized cloud hardware. The output
-is then sanity checked manually by several people. Based on that, the final gas
-parameter value is determined. Usually it will be the higher output of the two
-metrics rounded up. More details on the process will be added to this document
-in due time.
+For a practical guide on how to run the estimator, please take a look at
+[Running the Estimator](/practices/workflows/gas_estimations.html) in the
```

Comment:
In https://github.com/near/nearcore/blob/ca284e1e73078c7e61f0ee5d12e00b500642fb3a/docs/architecture/gas/estimator.md, this link is dead.  I wonder if we need to do the following or it's dead because the PR is not merged yet.
```suggestion
[Running the Estimator](../../practices/workflows/gas_estimations.html) in the
```

Reply:
seems to work now, with the `.md ` extension and relative paths it seems to work locally and also in the PR